src/lang-promql/complete/promql.terms.ts

```diff
@@ -18,66 +18,254 @@ export const binOpTerms = [
 ];
 export const binOpModifierTerms = [{ label: 'on' }, { label: 'ignoring' }, { label: 'group_left' }, { label: 'group_right' }];
 export const functionIdentifierTerms = [
-  { label: 'abs' },
-  { label: 'absent' },
-  { label: 'absent_over_time' },
-  { label: 'avg_over_time' },
-  { label: 'ceil' },
-  { label: 'changes' },
-  { label: 'clamp_max' },
-  { label: 'clamp_min' },
-  { label: 'count_over_time' },
-  { label: 'days_in_month' },
-  { label: 'day_of_month' },
-  { label: 'day_of_week' },
-  { label: 'delta' },
-  { label: 'deriv' },
-  { label: 'exp' },
-  { label: 'floor' },
-  { label: 'histogram_quantile' },
-  { label: 'holt_winters' },
-  { label: 'hour' },
-  { label: 'idelta' },
-  { label: 'increase' },
-  { label: 'irate' },
-  { label: 'label_replace' },
-  { label: 'label_join' },
-  { label: 'ln' },
-  { label: 'log10' },
-  { label: 'log2' },
-  { label: 'max_over_time' },
-  { label: 'min_over_time' },
-  { label: 'minute' },
-  { label: 'month' },
-  { label: 'predict_linear' },
-  { label: 'quantile_over_time' },
-  { label: 'rate' },
-  { label: 'resets' },
-  { label: 'round' },
-  { label: 'scalar' },
-  { label: 'sort' },
-  { label: 'sort_desc' },
-  { label: 'sqrt' },
-  { label: 'stddev_over_time' },
-  { label: 'stdvar_over_time' },
-  { label: 'sum_over_time' },
-  { label: 'time' },
-  { label: 'timestamp' },
-  { label: 'vector' },
-  { label: 'year' },
+  {
+    label: 'abs',
+    info: 'Returns the input vector with all sample values converted to their absolute value.',
+  },
+  {
+    label: 'absent',
+    info:
+      'Returns an empty vector if the vector passed to it has any elements and a 1-element vector with the value 1 if the vector passed to it has no elements. This is useful for alerting on when no time series exist for a given metric name and label combination.',
```

Comment:
on this point, I think the simpler should be to use what @juliusv shared. At least for the info that are quite long. But maybe Julius will have a different opinion.
It wasn't super clear for me if we should take as is what Grafana is proposing

Reply:
After reading and trying both I strongly believe @juliusv 's version is much better as "help". It's simpler and has the right length. I made a commit to use Julius's version 7f23ed9ee5932498dc4d55af4b3628001c4fffb2 If he accepts this then we are good, or else I will reverse back to the Grafana and try to shorten them